layouts/partials/tabs.html

```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
+        <div class="grid-container">
+            <div>
+               <img src="images/content_images/ds-landscape.png" alt="ds-landscape" align="left">
+            </div>
+            <div>
+            <p>
+	      Data Science makes it possible to analyze massive amounts of data
+	      and gain meaningful insights. 
+            </p>
+	    <p>
+              Python has a rich ecosytem of libraries that enable Data Science
+	      workflows. These workflows typically involve the following key
+	      processes:
+                <ul class="content-tab">
+                <li><b>Data Extraction:</b> Pandas, Beautiful Soup</li>
+                <li><b>Data Transformation:</b> Pandas</li>
+                <li><b>Data Manipulation:</b> Seaborn, Matplotlib</li>
+                <li><b>Data Exploration:</b> Scikit-learn, SciPy</li>
+                <li><b>Data Presentation:</b> Matplotlib and <a href="./index.html/#tab-visualisation" target="_blank" rel="noreferrer">several others</a></li>
+              </ul>
+            </p>
+            </div>
+	</div>
+	<p>
+	  All data manipulation begins with gathering or extracting data. <a 
+	  href="https://pandas.pydata.org"> Pandas </a> helps in data discovery
+          and handling, while <a href=
+	  "https://www.crummy.com/software/BeautifulSoup/">Beautiful Soup</a>
+	  helps in gathering and web-scrapping data sets.
+	</p>
+	<p>
+          Before machine learning algorithms can be applied to data, it requires
+	  cleaning and preparation by tools such as Pandas. <a href=
+          "https://seaborn.pydata.org">Seaborn</a> is well known for its <a href=
+	  "https://towardsdatascience.com/how-to-perform-exploratory-data-analysis-with-seaborn-97e3413e841d">
+	  exploratory data analysis (EDA)</a> capabilities, while some of the
+          backbone processes required for machine learning, namely, regression
+	  methods, classification, clustering, model validation and selection are
+	  well served by <a href="https://scikit-learn.org">Scikit-learn</a> and
+	  <a href="https://scikit-learn.org">Scipy</a> (statistical computing).
+	</p>  
+        <p>
+	And of course, <b>NumPy</b> is the foundation of almost all of these tools
+	  used for Data Science.
+        </p>
+	<p>
+	Besides these, <a href="http://keras.io">Keras</a> offers deep learning
+	with neural networks. <a href="https://www.tensorflow.org">Tensorflow</a>
+	framework helps to define and run computations involving tensors that have
+	broad applications in Speech and image recognition, Text-based
+	applications, Time-series analysis and Video detection.
+	</p>
+	<div class="grid-container">
+            <div>
+	    <p>
+              Effective data analytics require deep knowledge of the data domain
+              (e.g., Retail, Healthcare, Marketing, Finance, Social Media,
+	      Automation, Sales, Travel, Pure Science, Aeronautics etc.) as well
```

Comment:
I would stop after `Travel` and discard the rest of this paragraph, to keep the text to a reasonable length.

Reply:
And then perhaps finish off with something like: "Effective Data Science workflows typically involve tools like MLflow for experiment hyperparameter and result tracking, and DVC for versioning data".

Reply:
Included. Ref https://github.com/numpy/numpy.org/commit/9bbc977facda006ee1cfa3d166fd359be75c3979